privaterelay/templates/premium-promo.html

```diff
@@ -0,0 +1,254 @@
+            <p>{% ftlmsg 'premium-promo-hero-body-html' monthly_price=monthly_price %}</p>
+            <div class="premium-promo-hero-cta">
+                {% if premium_available_in_country %}
+                    <a
+                        class="mzp-c-button mzp-t-product "
+                        href="{% premium_subscribe_url accept_language country_code %}"
+                        data-ga="send-ga-pings"
+                        data-event-category="Purchase Button"
+                        data-event-label="premium-promo-cta"
+                        data-cookie-for-server="clicked-purchase"
+                    >
+                        {% ftlmsg 'premium-promo-hero-cta' %}
+                    </a>
+                {% else %}
+                    <button
+                        disabled="disabled"
+                        title="{% ftlmsg 'premium-promo-availability-warning' %}"
+                        class="mzp-c-button mzp-t-product"
+                    >
+                        {% ftlmsg 'premium-promo-hero-cta' %}
+                    </button>
+                {% endif %}
+            </div>
+            <p class="premium-promo-availability-warning">{% ftlmsg 'premium-promo-availability-warning' %}</p>
+        </div>
+        <div class="has-hero-image">
+            <img alt="" src="{% static 'images/newlanding/a/hero-image-bg.svg' %}" class="c-hero-image--bg"/>
+            <img alt="" src="{% static 'images/newlanding/a/hero-image-premium.svg' %}" class="c-hero-image"/>
+            {% comment %}
+                TODO: Find a way to localise the foreground content, or hide the callouts in languages other than English.
+            {% endcomment %}
+            <img alt="" src="{% static 'images/newlanding/a/hero-image-fg.svg' %}" class="c-hero-image--fg"/>
+        </div>
+    </section>
+
+    <section>
+        <div class="mzp-l-content">
+        <div class="perks-content">
+        <h2 class="perks-headline">{% ftlmsg 'premium-promo-perks-headline' %}</h2>
+        <span class="perks-lead">{% ftlmsg 'premium-promo-perks-lead' %}</span>
+        <div class="perk">
+        <img src="{% static 'images/premium-promo/illustration-unlimited.svg' %}" class="perk-illustration">
+        <div class="perk-description">
+            <h3 class="perk-headline ff-Met">{% ftlmsg 'premium-promo-perks-perk-unlimited-headline' %}</h3>
+            <p class="perk-body">
+                {% ftlmsg 'premium-promo-perks-perk-unlimited-body' %}
+            </p>
+            {% if premium_available_in_country %}
+                <a
+                    class="mzp-c-button mzp-t-product "
+                    href="{% premium_subscribe_url accept_language country_code %}"
+                    data-ga="send-ga-pings"
+                    data-event-category="Purchase Button"
+                    data-event-label="premium-promo-perk-cta"
```

Comment:
Should we differentiate between each perk section?

Reply:
Likewise: e769db18